src/cipher/random.ml

```diff
@@ -8,7 +8,46 @@
 (* TODO LATER Is there something with lighter dependencies? Although perhaps
    these are not so bad... *)
 
+exception Entropy_is_not_initialized
```

Comment:
We don't need this exception. We can just use `failwith` in the `None` case, or even `assert false`. It should be unreachable because of program composition, so we just need some fatal error that will either point people to the source line, or include a message about what it is, if we ever mess up.